### ISSUE TITLE
Empty order-complete page when checkout and cart pages point to woocommerce blocks pages

### ISSUE THREAD

**Issue by Shazgu** [woocommerce/woocommerce-gutenberg-products-block#3194]
## Describe the bug
I'm experiencing an issue that's been reported by various people but there doesn't seem to be a solution. 
It’s the bug where the order-complete page comes up blank even though the order is processed ok. See screenshot.

There are no issues found in console - the page just doesn't show any content apart from the page heading.
It’s not a custom page. We have it set up to the default order-complete page.

I’ve spent the day turning things on and off to find out what’s causing it. I first thought that in my case it was caused by Elementor, because when I disabled it the page content appeared again. Turning it on again brought the problem back. (I can’t remove the plugin because it messes up the site. Large chunks of it are built with Elementor). 
I then discovered  that even with Elementor active, it's actually the page settings for checkout and cart. When checkout and cart are set in woocommerce > Settings > Advanced to the default woocommerce pages, the problem goes away. Setting them to the woocommerce blocks pages brings the issue back. 
(From what I see others reporting, this issue is triggered quite often but by different plugins, themes.)

Hope you can help.

## Screenshots
<img width="1669" alt="empty order complete page" src="https://user-images.githubusercontent.com/71322363/94200478-6330d580-feb2-11ea-9e74-f7a87941a57b.png">

### WordPress (please complete the following information):
* Core version: 5.5.1
* Gutenberg version (if installed): -
* WooCommerce version: 4.5.2
* WooCommerce Blocks version: 3.4.0
* Site language: English
* Any other plugins installed: Elementor, Stripe and many more

### Desktop (please complete the following information):
* OS: Happens on different OSs
* Browser Happens in different browsers

**Comment by Shazgu** [comment]
UPDATE:
We've managed to solve it. It's a clash between the checkout block and Elementor.
I narrowed the issue down to the blocks checkout page. 
I then saw that the page had switched itself to the Elementor editor (Not sure how). I clicked to return to the WordPress editor. The checkout block came out broken (it was asking whether to convert to a standard text block or html). I clicked the option to try to resolve and it restored the blocks checkout block and everything started working properly again.
Interestingly, the blocks cart page had been edited with Elementor but was working fine.

I hope this is helpful to others who are experiencing the same problem.
